### ISSUE TITLE
Using Character Controller on Server

### ISSUE THREAD

**Issue by ryansalatti** [ssatguru/BabylonJS-CharacterController#71]
I was looking at the [Use characterController for secondary players #64](https://github.com/ssatguru/BabylonJS-CharacterController/issues/64) issue, which seems similar to what I'm trying to do. I'm setting up a server to verify movement, was curious if there was a way for this to work when the avatar meshes are just boxes without animations.

Thank you!

**Comment by ssatguru** [comment]
@ryansalatti Yes , it should work without animations. Give it a try and let me know if you run into any issues.
